src/common/chat/Tokenizer.ts

```diff
@@ -8,6 +8,7 @@ import type {
 	VoidToken,
 } from "@/common/chat/ChatMessage";
 import { Regex } from "@/site/twitch.tv";
+import { parse } from "tldts";
```

Comment:
nit: make this a default import

```suggestion
import tldts from "tldts";
```